src/linker/runner.py

```diff
@@ -89,7 +96,7 @@ def get_environment_args(config: Config) -> List[str]:
             "--default-resources",
             f"slurm_account={resources['account']}",
             f"slurm_partition='{resources['partition']}'",
-            f"mem={resources['memory']}",
+            f"mem_mb={resources['memory']*1024}",
```

Comment:
This block looks very similar to the block above in `rule.py`. Is there a way to reduce the repetition?

Reply:
I simplified the `runner` one by changing the keys in the config, but for mysterious reasons the `rule` change causes a syntax error in snakemake, so I'm going to leave it.